ibis/tests/all/test_window.py

```diff
@@ -323,25 +329,120 @@ def test_bounded_preceding_windows(backend, alltypes, df, con, window_fn):
         ),
     ],
 )
+@pytest.mark.parametrize(
+    ('ordered'), [param(True, id='orderered'), param(False, id='unordered')],
+)
 @pytest.mark.xfail_unsupported
-def test_unbounded_window(backend, alltypes, df, con, result_fn, expected_fn):
+def test_unbounded_window_grouped(
+    backend, alltypes, df, con, result_fn, expected_fn, ordered
+):
     if not backend.supports_window_operations:
         pytest.skip(
             'Backend {} does not support window operations'.format(backend)
         )
 
-    expr = alltypes.mutate(
-        val=result_fn(
-            alltypes,
-            win=ibis.window(
-                group_by=[alltypes.string_col], order_by=[alltypes.id],
-            ),
+    # Define a window that is
```

Comment:
can you make this a fixture instead? might be easier to understand (and you are using it again below)

Reply:
The other usage of `ordered` is actually a little different because it skips some backends depending on the value of `ordered`. I think doing that requires `@pytest.mark.parametrize` (and I feel if that's the case we might as well use `@pytest.mark.parametrize` for this occurrence of `ordered` as well)
https://github.com/ibis-project/ibis/blob/04d9cbe6b1577d5725972705753942f2231967d1/ibis/tests/all/test_window.py#L399-L419
Would you be OK with keeping this as is?